setup.py

```diff
@@ -1,29 +0,0 @@
-"""Setup script for HaikuPorter
```

Comment:
I think we actually use this to package HaikuPorter? Not sure...

Reply:
setup.py is not needed and can be safely replaced by newer solution which is using pyproject.toml to install the package. Installing package with pip works flawlessly. I'll link article/document describing why setup.py is deprecated and how should it be replaced.

Reply:
And the links: 
1. https://peps.python.org/pep-0517/
2. https://peps.python.org/pep-0518/
3. https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html#summary